src/eventTile.ts

```diff
@@ -69,19 +72,34 @@ const handleEventCreationClick = (event: MouseEvent): void => {
     }
 };
 
-const placeNewEventTile = (currentEventTile: HTMLElement, eventData: EventData): void => {
-    const eventTileTitle: HTMLElement = currentEventTile.querySelector(".event-tile-title");
+const placeNewEventTile = (currentEventTile: HTMLElement, eventData: EventData) => {
+    const eventTileTitle: HTMLElement | null = currentEventTile.querySelector(".event-tile-title");
```

Comment:
Isn't querySelector returning `HTMLElement | null` already?
```suggestion
    const eventTileTitle = currentEventTile.querySelector(".event-tile-title");
```

Reply:
it gives me "Element | null" and afterwards i  could not select innerText from this elements, it have me error : Property 'innerText' does not exist on type 'Element'